cmd/ocsp-updater/main_test.go

```diff
@@ -31,10 +31,24 @@ func (ca *mockCA) GenerateOCSP(xferObj core.OCSPSigningRequest) (ocsp []byte, er
 }
 
 type mockPub struct {
-	sa core.StorageAuthority
+	sa     core.StorageAuthority
+	double bool
 }
 
 func (p *mockPub) SubmitToCT(_ []byte) error {
+	if p.double {
+		err := p.sa.AddSCTReceipt(core.SignedCertificateTimestamp{
+			SCTVersion:        0,
+			LogID:             "another-id",
```

Comment:
Or, more to the point, why introduce the double variable at all? Why not always submit to two logs? Are there other tests that fail?